pyanaconda/core/constants.py

```diff
@@ -103,7 +103,7 @@
 GRAPHICAL_TARGET = 'graphical.target'
 
 # Network
-NETWORK_CONNECTION_TIMEOUT = 45  # in seconds
+NETWORK_CONNECTION_TIMEOUT = 46  # in seconds
```

Comment:
The explanation should be documented in a comment above this constant in case that we will need to modify it later.

Reply:
Good point.